src/applications/simple-forms/mock-simple-forms-patterns/components/ArrayBuilderSummaryCardList.jsx

```diff
@@ -0,0 +1,173 @@
+/**
+ * For array builder pattern
+ * Shows a title and cards with edit and remove links
+ */
+import React, { useState } from 'react';
+import { Link } from 'react-router';
+import { VaModal } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { setData } from 'platform/forms-system/src/js/actions';
+import get from 'platform/utilities/data/get';
+import set from 'platform/utilities/data/set';
+
+const ArrayBuilderSummaryCardList = ({
+  editBasePath,
+  formDataPath,
+  setFormData,
+  formData,
+  CardContent,
+  title,
+  titleHeaderLevel = '3',
```

Comment:
This component also shows up on the Review-page, and a static prop-value here causes a duplicate-H3 a11y-bug there.  I think this needs to be dynamic based on where it's rendered.  [See also my comment below on mapStateToProps]

Reply:
Keep in mind that this Review-page dupe-H3 a11y-bug might be better handled within the Review-page instead.

Reply:
Good callout, I tried to look into a quick fix by adding form context, but apparently we don't have access to it in redux here, and maybe like you mentioned maybe we can look into handling it on the review page instead.  We have a specific ticket to look at all the review stuff, I've mentioned this there, and we can handle it in that ticket: https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/904

Reply:
Sounds good.  Thanks!